WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -1221,7 +1234,46 @@ extension NotificationDetailsViewController: SuggestionsTableViewDelegate {
     }
 }
 
+// MARK: - Milestone notifications
+//
+private extension NotificationDetailsViewController {
+
+    /// Determines if the notification content is a views milestone
+    var isViewMilestone: Bool {
+        if let type = note.type, type.contains("view_milestone"), FeatureFlag.milestoneNotifications.enabled {
+            return true
+        }
+        return false
```

Comment:
I think we can simplify this a bit, what do you think?

We can also change the `contains` to an equals since we know the exact type the notification will be. 

```suggestion
        return FeatureFlag.milestoneNotifications.enabled && note.type == "view_milestone"
```